src/index.ts

```diff
@@ -0,0 +1,10 @@
+import getSdk = require('resin-sdk');
+import settings = require('resin-settings-client');
+
+export = getSdk({
+	apiUrl: settings.get('apiUrl'),
+	imageMakerUrl: settings.get('imageMakerUrl'),
+	dataDirectory: settings.get('dataDirectory'),
+	apiVersion: 'v2',
```

Comment:
Excellent point. Fixed - I've dropped the explicit version, and left this for the SDK to handle this for itself.